docs/changes/coordinates/14675.api.rst

```diff
@@ -0,0 +1 @@
+Removed deprecated angle utilities and orbital elements (``calc_moon``).
```

Comment:
I didn't realize it wasn't mentioned before. Like this, @eerovaher ?

```suggestion
Removed deprecated angle utilities and orbital elements.
```

Reply:
I'd rather remove the change log entry entirely.

Reply:
To remove over 300 lines of code without any change log? I am not sure if I am comfortable with that. Are you sure absolutely no one is somehow still using this?

Reply:
Despite not having a name that begins with an underscore this function was considered to be private and was not documented, see https://docs.astropy.org/en/v4.3.1/search.html?q=calc_moon

Reply:
Still, the angle utilities being removed need change log?

Reply:
Accepted your suggestion. Thanks!